templates/commands/render/render_action_foreach.go

```diff
@@ -34,7 +35,7 @@ func actionForEach(ctx context.Context, fe *model.ForEach, sp *stepParams) error
 			return err
 		}
 	} else {
-		if err := celCompileAndEval(ctx, sp.scope, *fe.Iterator.ValuesFrom, &values); err != nil {
+		if err := utils.CelCompileAndEval(ctx, sp.scope, *fe.Iterator.ValuesFrom, &values); err != nil {
 			return err
```

Comment:
Regarding the lint error that's triggered on this line:

```
Check failure on line 39 in templates/commands/render/render_action_foreach.go

GitHub
Actions
/ go_lint / golangci

error returned from external package is unwrapped: sig: func github.com/abcxyz/abc/templates/utils.CelCompileAndEval(ctx context.Context, scope *github.com/abcxyz/abc/templates/utils.Scope, expr github.com/abcxyz/abc/templates/model.valWithPos[string], outPtr any) error (wrapcheck)
```

You can add `//nolint:wrapcheck` at the end of the line to silence it. This is appropriate beacuse CelCompileAndEval returns an error that doesn't need to be wrapped.

Reply:
Thank you!